sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/MetadataReferences/AspNetCoreMetadataReference.cs

```diff
@@ -18,7 +18,7 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-#if NETCOREAPP
+#if NET5_0
```

Comment:
In more files:
```suggestion
#if NET
```

Reply:
I'm not sure about this. It is not in documentation and it seems that [they didn't took a decision yet](https://github.com/dotnet/docs/pull/21556#issuecomment-726853578). When can change this later when we will know more about it.